source/objects/lander.lua

```diff
@@ -80,9 +80,9 @@ local function moveShip(lander, dt)
 	lander.y = lander.y + lander.vy
 
 	-- Set left boundary
```

Comment:
I'd say we have to spawn the player more right and set the boundary so that no end of the terrain cannot be seen.

Reply:
Removing the edge of the world means removing the visual cue that you came actually move left.

Would it be better to remove the flat terrain at the start, permit a tiny bit of left movement, then influence the wall?

The player needs feedback so let's think about what's best.

Reply:
Maybe at some point we can add some graphics like a city scape that makes it obvious that moving left isn't an option.
I'd say lets keep it for now. I'll try to think about some visuals for this scenario.